frontend/src/routes/Applications/ApplicationsPage.tsx

```diff
@@ -1,37 +1,64 @@
 /* Copyright Contributors to the Open Cluster Management project */
 
 import { AcmPage, AcmPageHeader, AcmSecondaryNav, AcmSecondaryNavItem } from '../../ui-components'
-import { Fragment, lazy, Suspense, useEffect } from 'react'
-import { Link, Redirect, Route, Switch, useLocation } from 'react-router-dom'
+import { Fragment, lazy, Suspense, useEffect, useState } from 'react'
+import { Link, matchPath, Redirect, Route, Switch, useLocation, useRouteMatch } from 'react-router-dom'
 import { useSetRecoilState } from 'recoil'
 import { discoveredApplicationsState, discoveredOCPAppResourcesState } from '../../atoms'
 import { useTranslation } from '../../lib/acm-i18next'
 import { queryRemoteArgoApps, queryOCPAppResources } from '../../lib/search'
 import { useQuery } from '../../lib/useQuery'
 import { NavigationPath } from '../../NavigationPath'
+import { LoadingPage } from '../../components/LoadingPage'
 
 const ApplicationsOverviewPage = lazy(() => import('./Overview'))
 const AdvancedConfigurationPage = lazy(() => import('./AdvancedConfiguration'))
 
 export default function ApplicationsPage() {
-    const location = useLocation()
     const { t } = useTranslation()
+    const location = useLocation()
+    const applicationsMatch = useRouteMatch()
+    const advancedMatch = matchPath(location.pathname, NavigationPath.advancedConfiguration)
 
-    const { data, startPolling } = useQuery(queryRemoteArgoApps)
-    const dataOCPResources = useQuery(queryOCPAppResources).data
-    const startPollingOCPResources = useQuery(queryOCPAppResources).startPolling
+    const { data, loading, startPolling } = useQuery(queryRemoteArgoApps)
+    const {
+        data: dataOCPResources,
```

Comment:
Provide a new name for the duplicated properties from `useQuery` rather than calling `useQuery` multiple times for the same query.